src/check_update.c

```diff
@@ -63,11 +66,30 @@ enum swupd_code check_update()
 		ret = SWUPD_SERVER_CONNECTION_ERROR;
 	}
 
+	if (log_get_level() >= globals.verbose) {
+		/* Retrieve latest_format */
+		/* Sending latest server_version to get the latest format */
+		int temp_server_format = get_server_format(server_version);
+		if (temp_server_format > 0) {
+			sprintf(latest_format, "%d", temp_server_format);
+		}
+
+		/* Retrieve current format */
+		int temp_current_format = get_current_format();
+		if (temp_current_format > 0) {
+			sprintf(current_format, "%d", temp_current_format);
+		}
+	}
+
 	if (current_version > 0) {
-		info("Current OS version: %d\n", current_version);
+		info("Current OS version: %d", current_version);
+		info_verbose(" (format %s)", current_format);
```

Comment:
you can move the logic of the printf here, so you don't need to calculate and store and then print. 

		int temp_current_format = get_current_format();
		if (temp_current_format > 0) {
			info_verbose(" (format %d)", temp_current_format);
		} else {
			info_verbose(" (format unknown)", temp_current_format);
		}

Reply:
Make sense...remove that extra sprintf